### ISSUE TITLE
Auto levels calculating incorrect value

### ISSUE THREAD

**Issue by StuartNicholls** [nachocinalli/adapt-lens#1]
Increments from _ariaLevel if it exists. AriaLevel is 0 (zero) by default, which will then report the heading level incorrectly/ 

Affects corses from exported from the authoring tool.

Suggest updating check for 0

**Comment by nachocinalli** [comment]
Thanks, it seems like it was fixed on the recent releases of the core.
[v6.20.2](https://github.com/adaptlearning/adapt-contrib-core/releases/tag/v6.20.2) 
[v6.20.3](https://github.com/adaptlearning/adapt-contrib-core/releases/tag/v6.20.3)

It looks that the [Accordion component](https://github.com/adaptlearning/adapt-contrib-accordion/blob/master/templates/accordion.jsx) has the same issue.

I will test again when the framework is updated.

**Comment by StuartNicholls** [comment]
Sorry, yes I should have mentioned that. Core had an update for a more complete version of aria-level control. The update causes an issue with sub-headings in components. This only affects JSX temperates and when _ariaLevel is set to 0.  

Currently, sub headings increment from _ariaLevel if it exists. So if set to 0 - which is the falsey value - subheadings will  always render at level 1.

I was actually updating the accordion and the list component on behalf of Kineo, so I thought mention here also.

**Comment by nachocinalli** [comment]
Thank you very much for the explanation.
